aioftp/server.py

```diff
@@ -481,23 +481,31 @@ async def write_response(self, stream, code, lines="", list=False):
                 await write(code + "-" + line)
             await write(code + " " + tail)
 
-    async def parse_command(self, stream):
+    async def parse_command(self, stream, censor_commands=("pass",)):
         """
         :py:func:`asyncio.coroutine`
 
         Complex method for getting command.
 
-        :param stream: connection steram
+        :param stream: connection stream
         :type stream: :py:class:`asyncio.StreamIO`
 
+        :param censor_commands: An optional list of commands to censor.
+        :type censor_commands: :py:class:`tuple` of :py:class:`str`
+
         :return: (code, rest)
         :rtype: (:py:class:`str`, :py:class:`str`)
         """
         line = await stream.readline()
         if not line:
             raise ConnectionResetError
         s = line.decode(encoding=self.encoding).rstrip()
-        logger.info(s)
+
+        if s[:4].lower() in censor_commands:
```

Comment:
Command directly resolved on 501 line of old version. I think it is better to move this logic with logging below.

Reply:
All right, I've moved it below the splitting thing, and made it more in line with the suggestion for `client.py`.